collections/deep_merge.ts

```diff
@@ -0,0 +1,116 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * Merges the two given Records, recursively merging any nested Records with the second collection overriding the first in case of conflict
+ *
+ * For arrays, maps and sets, a merging strategy can be specified to either "replace" values, or "concat" them instead.
+ * Use "includeNonEnumerable" option to include non enumerable properties too.
```

Comment:
Please wrap at 80 columns.

(I'm surprised `deno fmt --check` didn't catch this)

Reply:
it hasn't been caught since `deno fmt` didn't wrap them so I assume it's the same for `--check`